main.py

```diff
@@ -21,7 +24,23 @@
 
 PRODUCTION = False # added for convenience, all factories eventually created in main and passed to trade_bot
 
+# SERVER SETUP
+def flask_init():
+    endpoints = []
+
+    endpoints.append(Endpoint("/orders-open", get_open_orders))
+    endpoints.append(Endpoint("/orders-closed", get_closed_orders))
+
+    ws = WebService(__name__, endpoints)
+    ws.run()
+
 def main():
+
+    # Avoid blocking main thread
+    flask_thread = threading.Thread(target=flask_init)
+    flask_thread.daemon = True
+    flask_thread.start()
```

Comment:
can we start its thread inside a WebService method?

https://github.com/BensGitHub2022/MT5-Expert-Advisor/blob/ben/webservice_example/src/web_service/web_service.py

see the webservice.py example